### PR TITLE
Spelling, Links, and PSX Core Options

### PR DESCRIPTION
**Pull request by alexknop**
-made spelling changes to fix typos or make capitalization of titles consistent
-made some hyperlinks more obvious they are hyperlinks
-updated process for enabling dualshock controlelrs in RetroArch Core Options for PSX

### CODE REVIEW

File: docs/Playstation-1.md [hunk 1]
```diff
@@ -200,15 +200,16 @@ For a more complete resource, please check the [PlayStation DataCenter](http://p
 
 ### Multitap (3-5 Players)
 
-lr-pcsx_rearmed has support for multitap, but not all games read input when a multitap is connected, so a per-ROM Core Options file should be created for multitap compatible games. Set the Core Options for **Pad 3 Type**, **Pad 4 Type** (and so on, depending on how many players are supported by the game) to the relevant Controller Type that the game supports. See [Setting Core Options per-ROM](RetroArch-Core-Options#setting-core-options-per-rom).
+lr-pcsx_rearmed has support for multitap, but not all games read input when a multitap is connected, so a per-ROM Core Options file should be created for multitap compatible games. Set the Core Options for **Pad 3 Type**, **Pad 4 Type** (and so on, depending on how many players are supported by the game) to the relevant Controller Type that the game supports. See [Setting Core Options per-ROM](
+-Core-Options#setting-core-options-per-rom).
```
Comment (cmitu) [inline docs/Playstation-1.md:204]: There's an extra newline here.

**Comment by cmitu** [comment]
Thank you for the contribution. 
The spelling/capitalisation/naming fixes are welcomed, but the change of the links to `**(link)**` is not - is not how the links are used in the Wiki. Can you adjust the links back to their previous style ?

**Comment by alexknop** [comment]
Hello,
I can certainly understand adding **(link)** is not wanted, however can I change some of these to bold and/or underline? Having them in plaintext easily blends in with the rest of the text. I was just looking to make the hyperlinks more obvious, and am open to doing it another way.

**Comment by cmitu** [comment]
You can add a **bold** styling to them, but I think the rest (color/text decoration) should be handled by the site's CSS.
I think the CSS of the site doesn't emphasise links enough, we might have lost that during the transition from Github Wiki to the new docs repository.

**Comment by bspeakmon** [comment]
My PR #48 fixes the link coloring without messing with the CSS or the link style.

**Comment by alexknop** [comment]
Just made the suggested fixes. I'm ready for you guys to review again

**Comment by cmitu** [comment]
Looks fine, thank you !

**Pull request merged**